tools/generate-countries.ipynb

```diff
@@ -996,9 +1200,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.7",
    "language": "python",
-   "name": "python3"
+   "name": "python3.7"
   },
```

Comment:
Should leave the kernel as the default `python` one. Changing it to a specific kernel like `python3.7` means the notebook won't run for others unless they also have that kernel.